aea/cli/utils/click_utils.py

```diff
@@ -196,3 +196,37 @@ def handle_parse_result(
             )
 
         return super().handle_parse_result(ctx, opts, args)
+
+
+def password_option(confirmation_prompt: bool = False, **kwargs) -> Callable:  # type: ignore
+    """Decorator to ask for input if -p flag was provided or use --password to set password value in command line."""
+
+    def callback(ctx, _, value: bool) -> bool:  # type: ignore
+        if value is True:
+            ctx.params["password"] = ctx.params.get("password") or click.prompt(
+                "Enter password please",
```

Comment:
```suggestion
                "Enter password",
```

(this is a very picky comment... feel free to ignore) I think `please` can be avoided here.

Consider adding a semicolon `:` if it is not already added by Click.